src/inmanta/loader.py

```diff
@@ -662,3 +666,48 @@ def module_in_prefix(module: types.ModuleType, prefix: str) -> bool:
     for mod_name in loaded_modules:
         del sys.modules[mod_name]
     importlib.invalidate_caches()
+
+
+@dataclass(frozen=True)
+class PythonModule:
+    name: str
+    version: str
+    files_in_module: list["SourceInfo"]
+
+
+class SourceInfo(BaseModel):
```

Comment:
I don't recall why I turned this into a pydantic model, this might not be necessary.